co_simulation/validation/dam_break_flex_wall/source/ProjectParametersCoSim.json

```diff
@@ -0,0 +1,154 @@
+{
+    "problem_data" :
+    {
+        "start_time" : 0.0,
+        "end_time" : 1.0,
+        "echo_level" : 0,
+        "print_colors" : true,
+        "parallel_type" : "OpenMP"
+    },
+    "solver_settings" :
+    {
+        "type" : "coupled_solvers.gauss_seidel_strong",
+        "echo_level" : 0,
+        "num_coupling_iterations" : 20,
+        "convergence_accelerators" : [
+            {
+                "type"      : "aitken",
+                "solver"    : "pfem",
+                "data_name" : "vel"
+            },
+            {
+                "type"      : "aitken",
+                "solver"    : "pfem",
+                "data_name" : "disp"
+            }
+        ],
+        "convergence_criteria" : [
+            {
+                "type"          : "relative_norm_previous_residual",
+                "solver"        : "structure",
+                "data_name"     : "vel",
+                "abs_tolerance" : 5e-6,
+                "rel_tolerance" : 5e-6
+            }
+        ],
+        "coupling_operations" : {
+            "compute_boundary_force_process": {
+                "type"       : "compute_boundary_force",
+                "echo_level" : 2,
+                "solver"            : "pfem",
+                "model_part_name"   : "PfemFluidModelPart.InterfaceWalls",
+                "write_output_file" : false,
+                "width"             : 1,
+                "interval"               : [0.0,"End"],
+                "output_file_settings": {
+                    "output_path": "force_output"
+                    }
+            },
+            "reset_pfem_kinematics_fluid": {
+                "type"       : "reset_pfem_kinematics",
+                "echo_level" : 3,
+                "solver"            : "pfem",
+                "model_part_name"   : "PfemFluidModelPart.Fluid",
+                "interval"               : [0.0,"End"]
+            }
+        },
+        "data_transfer_operators" : {
+            "mapper_1" : {
+                "type" : "kratos_mapping",
+                "mapper_settings" : {
+                    "mapper_type" : "nearest_neighbor"
```

Comment:
is it not possible to use the nearest element mapper because you don't have elements in on of the ModelParts?

Reply:
It should be possible. I used the nearest neighbor because I created the structure ModelPart nodes matching exactly the fluid ModelPart. The element types are different, the solid being triangles, the fluid being lines.